extra/1-currency-conversion.js

```diff
@@ -5,7 +5,10 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
+return price*1.4;
```

Comment:
Hi Adem, well done, you have done really great work. just one thing to draw your attention is the indentation. return should be indented 4 space to right as below.
```
function convertToUSD(price) {
    return price*1.4;
}
```

Reply:
Thanks Askin I fixed it but w3schools declares 2 space instead of 4